### DISCUSSION TITLE
Backup to an external SSD?

### DISCUSSION THREAD

**Discussion by Crypto-Bee** [seedvault-app/seedvault#410] [Q&A] [answered]
Hello,
on the homepage it says that you can make the backup to a USB flash drive. It is clear that it works with a USB stick.
Can Seedvault also backup to an external SSD?
Does it work with exFAT file format?
Has anyone already tried this?

Have a nice day, 
Crypto-Bee

**Comment by chirayudesai** [comment, marked as answer]
Hi,

If your ROM supports exFAT for your device, it should work.

The way to test it is see if the SSD shows up in the default "Files" app, and if you can copy files to it. If that works, then it should work for backups as well.

**Reply by Crypto-Bee** [reply]
Thanks for the quick reply! :-)
